spartan/terraform/deploy-release/main.tf

```
@@ -52,6 +52,31 @@ resource "helm_release" "aztec-gke-cluster" {
     value = var.L1_DEPLOYMENT_MNEMONIC
   }
 
+  set {
+    name  = "ethereum.deployL1ContractsPrivateKey"
+    value = var.L1_DEPLOYMENT_PRIVATE_KEY
+  }
+
+  set {
+    name  = "bootNode.seqPublisherPrivateKey"
+    value = var.BOOT_NODE_SEQ_PUBLISHER_PRIVATE_KEY
+  }
+
+  set {
+    name  = "proverNode.proverPublisherPrivateKey"
+    value = var.PROVER_PUBLISHER_PRIVATE_KEY
+  }
+
+  set_list {
+    name  = "validator.validatorKeys"
+    value = var.VALIDATOR_KEYS
+  }
```

```suggestion
 dynamic "set_list" {
    for_each = length(var.VALIDATOR_KEYS) > 0 ? [1] : []
    content {
      name  = "validator.validatorKeys"
      value = var.VALIDATOR_KEYS
    }
  }
```

I'm pretty sure the code as written will break deployments with the `deploy-network` workflow because the new variables don't have a default value, so the terraform command will just hang awaiting input. But also, if we set the default values to be empty, we need a dynamic block like the suggestion so that we only set when the values are not empty so that we actually use the values in the underlying yaml file.